src/components/molecules/FormField/index.js

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import validate from '_utils/validate-inputs'
+import Input from '_atoms/Input'
+
+import styles from './styles.css'
+
+class FormField extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      errorValidation: '',
+    }
+    this.validate = validate({
+      isRequired: props.required,
+      fieldType: props.type,
+      fieldRules: props.validations,
+      customRules: props.customValidations,
+    })
+    this.handleFormValidation = props.handleFormValidation
+  }
+
+  applyValidation = value => {
+    const validation = this.validate(value)
+
+    this.setState({ errorValidation: validation })
+    this.handleFormValidation(this.props.field, !validation)
```

Comment:
Why you pass the negative of the validation to the handleForm? It shouldn't receive the correct validation information?